docs/contributing/index.md

```diff
@@ -93,14 +93,28 @@ A package configures the types it allows in a file `towncrier.toml` located at t
 The content of this file must include the following.
 
 -   A brief message that summarizes the changes in your contribution.
+    -   Use a narrative format, in the past tense, proper English spelling and grammar, complete sentences, and inline markup as needed.
+    -   If you fix a bug, write what was broken and is now fixed.
+    -   If you add or change a feature, write a summary of previous behavior and what it does now.
```

Comment:
Should we add something like "The focus should be on what is relevant to a user of the software, not implementation details like what code you changed (unless it's a change to a public API)." ?

Reply:
@davisagli here's my latest attempt f2766c1b751733c245e0cd730f67fdf3bcbd85cd